vespalib/src/vespa/vespalib/testkit/time_bomb.cpp

```diff
@@ -14,7 +15,7 @@ void bomb(Gate &gate, vespalib::duration timeout) {
             return;
         }
     }
-    size_t countdown = std::min(count_s(timeout), 5l);
+    size_t countdown = std::min(count_s(timeout), INT64_C(5));
```

Comment:
Which compiler combo is unhappy here ?

Reply:
Any compiler on Darwin, where int64_t is `long long int`.